fe/src/Post/css/SelectBox.css

```
@@ -1,29 +1,34 @@
 .select-box {
     display: flex;
+    border: solid 0.1vw #bbb;
```

순서가 다른거 같아요!
https://developer.mozilla.org/ko/docs/Web/CSS/border
```suggestion
    border: 0.1vw solid #bbb;
```